src/api/redux/reducers.js

```diff
@@ -135,6 +171,51 @@ const getAvailableMarketplaceTokens = createSelector(
     ),
 )
 
+// TODO: get this out of connected-react-router redux instead of straight from the URL
```

Comment:
actually, this can't be driven by airswap.js redux because the routing logic is handled by the app. so our choices are to do this with search params, or move the selector to the app level. what do you think @cloudonshore